### PR TITLE
too many results #80

### PR DESCRIPTION
**Pull request by oliveregger**

### CODE REVIEW

**Comment by oliveregger** [comment]
https://build.fhir.org/ig/ehealthsuisse/ch-epr-mhealth/branches/oe_pdq_80/qa.html

File: input/pagecontent/iti-78.md [hunk 1]
```diff
@@ -25,13 +25,22 @@ A mobile app wants to query patients, which participate in the Swiss EPR.
 The message semantics is the same as defined in [2:3.78.4.1.2](https://profiles.ihe.net/ITI/PDQm/ITI-78.html#2378412-message-semantics) with a restriction if there are more than 5 matches. If there are more than 5 matches, the result should return zero matches.
```
Comment (qligier) [inline input/pagecontent/iti-78.md:25]:
```suggestion
The message semantics is the same as defined in [2:3.78.4.1.2](https://profiles.ihe.net/ITI/PDQm/ITI-78.html#2378412-message-semantics) with a restriction if there are more than 5 matches. If there are more than 5 matches, the result should return zero matches with an OperationOutcome requesting more query parameters.
```

Without an OperationOutcome, an empty search result should be considered successful and the client may not understand that a query refinement is required.


https://www.hl7.org/fhir/search.html#errors:

> When the search fails, a server SHOULD return an [OperationOutcome](https://www.hl7.org/fhir/operationoutcome.html) detailing the cause of the failure. Note that an empty search result is not an error.

File: input/fsh/pdqm.fsh [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+* issue.severity = #warning 
+* issue.code = #incomplete
+* issue.details from ChPdqmMoreAttriburesRequested (required)
+
+Instance: PDQmResponseMoreAttributesRequested
+InstanceOf: ch-pdqm-moreattributesrequested
+Title: "PDQm OperationOutcome More Attributes Requested Example"
+Description: "An example on the OperationOutcome for indicating which additional attributes are requested to lower the results number."
+Usage: #example
+* issue[0].severity = #warning
```
[the remark this answers is not in the document]
Comment (oliveregger) [reply input/fsh/pdqm.fsh:62]: In XCPD it is handled as warning, that's why we consider it here also as a warning (the query worked)

**Comment by qligier** [comment]
@oliveregger Can I delete the branch?

**Pull request merged**